server.py

```diff
@@ -1,12 +1,28 @@
 import os
 import subprocess
 import sys
+import re
 
 from bot_python_sdk.configuration_service import ConfigurationService
 from bot_python_sdk.store import Store
+from bot_python_sdk.logger import Logger
 
 configuration_service = ConfigurationService()
 store = Store()
+LOCATION = 'Server'
+
+#Function to validate the given IP Address
+def isValid(ip):
+    regex = '''^(25[0-5]|2[0-4][0-9]|[0-1]?[0-9][0-9]?)\.(
+                25[0-5]|2[0-4][0-9]|[0-1]?[0-9][0-9]?)\.(
+                25[0-5]|2[0-4][0-9]|[0-1]?[0-9][0-9]?)\.(
+                25[0-5]|2[0-4][0-9]|[0-1]?[0-9][0-9]?)'''
+
+    if(re.search(regex, ip)):
```

Comment:
if ( boolean ) return true;
else: return false;

can just be
return boolean

Reply:
Good catch. Fixed it.